kolibri/plugins/coach/assets/src/views/plan/GroupsPage/index.vue

```diff
@@ -19,7 +19,10 @@
           />
         </div>
 
-        <CoreTable>
+        <CoreTable
+          :dataLoading="groupsAreLoading"
```

Comment:
In testing I'm not actually seeing the group while the loader is shown 🤷🏻

Reply:
I don't understand, could you please elaborate a bit more?

Reply:
> Maybe a v-if="!groupsAreLoading" on the tbody here would do it?

In any case, I think this won't harm anything so no problem to commit it

Reply:
Perhaps it'd make more sense to have this implemented rather in `CoreTable`, though? Not only to have a condition to show the loader when `dataLoading` is truthy but also to not show the table body in this case? I think that may prevent some glitches (maybe in regards to cached data?) by ensuring that table data is never displayed as long as `dataLoading` is truthy.

Reply:
In your screencast, when you go to the Group tab, "Group 1" is visible with the loader underneath it. I think the loader being there should hide the content on the table until it is done loading.

However, I was not able to replicate it and tried as best I could so I don't think it's a blocking issue at all.

> Perhaps it'd make more sense to have this implemented rather in CoreTable, though?

Yeah I think you're right there for sure. That might be a good follow-up issue as it'd require regression testing across a bunch of tables and this is good to go otherwise. What do you think?

Reply:
>In your screencast, when you go to the Group tab, "Group 1" is visible with the loader underneath it. I think the loader being there should hide the content on the table until it is done loading.
>
> However, I was not able to replicate it and tried as best I could so I don't think it's a blocking issue at all.

Ah yes, I see now. Yes, that's right. I think it might be related to the way I prepared the recording when I added a piece of code to delay loading - it's possible that I placed it in the wrong place as I couldn't reproduce it either right now. Or it could be related to some kind of cache - we can resolve that in a follow-up and do regression testing, yes. I will open the issue.

Reply:
https://github.com/learningequality/kolibri/issues/10993